packages/core/test/acceptance/bootstrapping/application.feature.md

```diff
@@ -23,6 +23,5 @@ const app = new Application({
 });
 
 // get metadata about the registered components
-console.log(app.find('component.*')); // [Bindings] should match the 4 components registered above
```

Comment:
It's not clear to me what is the value of having components (the exported objects) bound in the context. Component users should be accessing features provided by the component (controllers, providers, etc), not the component object itself.